src/main/kotlin/racingcar/controller/InputController.kt

```diff
@@ -0,0 +1,45 @@
+package racingcar.controller
+
+import racingcar.utils.MAX_ROUND_COUNT
+import racingcar.utils.MIN_ROUND_COUNT
+import racingcar.utils.RacingRuleValidator
+import racingcar.utils.TEXT_IN_LINE_DELIMITER
+import racingcar.utils.TextUtils
+import racingcar.view.InputView
+import racingcar.view.OutputView
+
+class InputController(
+    private val inputView: InputView = InputView(),
+    private val outputView: OutputView = OutputView(),
+    private val racingRule: RacingRuleValidator = RacingRuleValidator()
+) {
+
+    fun readCarNames(): List<String> {
+        outputView.printMessage(CAR_NAMES_REQUEST_MESSAGE)
+        val input = inputView.readCarNames()
+        val carNames = TextUtils.removeTextsBlank(input.split(TEXT_IN_LINE_DELIMITER))
+
+        require(racingRule.isValidateNotDuplicatedCarNames(carNames)) { DUPLICATED_CAR_NAME_ERROR_MESSAGE }
+
+        return carNames
+    }
+
+    fun readRoundCount(): Int {
+        outputView.printMessage(ROUND_COUNT_REQUEST_MESSAGE)
+        val input = inputView.readRoundCount()
+
+        require(input.toIntOrNull() != null) { NOT_INTEGER_TYPE_ERROR_MESSAGE }
+        require(racingRule.isValidateRoundCountBoundary(input.toInt())) { ROUND_COUNT_BOUNDARY_ERROR_MESSAGE }
+
+        return input.toInt()
+    }
+
+    companion object {
+        const val CAR_NAMES_REQUEST_MESSAGE = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분)."
```

Comment:
```suggestion
        private const val CAR_NAMES_REQUEST_MESSAGE = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분)."
```
클래스 내부에서만 사용한다면, private 접근자를 붙여보면 어떨까요? (이하 관련 내용 동일)

Reply:
감사합니다! private 접근자를 사용하였습니다!